### PR TITLE
Add initial color palette structure and next color tokens

### PR DESCRIPTION
**Pull request by sam-b-rose**

### CODE REVIEW

File: polaris-tokens/src/palettes.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+export const palettes = {
```
Comment (aaronccasanova) [inline polaris-tokens/src/palettes.ts:1]: **thought:** Should these be individual exports? e.g. `export const gray = {...}`

Comment (sam-b-rose) [reply polaris-tokens/src/palettes.ts:1]: We are wanting to only keep them "private" for now and only reference them through the semantic tokens. We may decide to open these up in the future, but safer to keep them protected for now.

Comment (aaronccasanova) [reply polaris-tokens/src/palettes.ts:1]: Makes sense!

**Comment by aaronccasanova** [comment]
@samrose3 Thanks for kicking this off! This PR was an awesome reference for the final implementation: https://github.com/Shopify/polaris/pull/8246

**Pull request closed without merging**